### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by rocky-truong**

### CODE REVIEW

**Comment by rocky-truong** [comment]
image

**Comment by rocky-truong** [comment]
image
image

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+    title: title,
+    photo: photo,
+    notes: notes,
+    entryId: data.nextEntryId++
+  };
+  data.entries.unshift(newestEntry);
+  $newEntry.reset();
+  $imageSrc.setAttribute('src', 'images/placeholder-image-square.jpg');
+  var newEntryJSON = JSON.stringify(data);
+  localStorage.setItem('journal-local-storage', newEntryJSON);
```
Comment (codingcodymiller) [inline js/main.js:30]: This should be being done in `data.js` before the page unloads. I recommend you review `javascript-local-storage` for a refresher on how to deal with saving content in localStorage, the code for this project will be nearly identical to that.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**